external/iotivity/iotivity_1.2-rel/resource/c_common/iotivity_config.h

```diff
@@ -17,7 +17,7 @@
 
 #define HAVE_FCNTL_H 1
 
-#define HAVE_MEMORY_H 1
+//#define HAVE_MEMORY_H 1
```

Comment:
will this break ARM build?

Reply:
There's no problem.
In arm-none-eabi toolchain, `memory.h` only includes `string.h` file.
and HAVE_STRING_H has already been defined in this iotivity_config.h at line 30.